src/main/java/net/atos/zac/app/taken/converter/RESTTaakHistorieConverter.java

```diff
@@ -69,8 +71,18 @@ private RESTTaakHistorieRegel convert(final HistoricTaskLogEntry historicTaskLog
 
     private RESTTaakHistorieRegel convertData(final HistoricTaskLogEntryType type, final String data) {
         return switch (type) {
-            case USER_TASK_CREATED -> new RESTTaakHistorieRegel(CREATED_ATTRIBUUT_LABEL);
-            case USER_TASK_COMPLETED -> new RESTTaakHistorieRegel(COMPLETED_ATTRIBUUT_LABEL);
+            case USER_TASK_CREATED -> new RESTTaakHistorieRegel(
+                    CREATED_ATTRIBUUT_LABEL,
```

Comment:
How is this first argument, the attribute label used? Should that really be `aangemaakt`?

Reply:
This is handled by the UI to show history table. Now localized